Update End User Resources (#4286)
open-telemetry/opentelemetry.io: content/en/community/end-user/_index.md

---

> title: End-user Resources

Reviewer: ```suggestion
title: End-user resources
```

> linkTitle: End User

Reviewer: ```suggestion
linkTitle: End user
```

description: Vendor-agnostic resources for OpenTelemetry users
---

The
[OpenTelemetry End User SIG](https://github.com/open-telemetry/community/tree/main/working-groups/end-user)
has heard feedback from users who desire a vendor-agnostic space to discuss
OpenTelemetry adoption and implementation, so we provide the following ways for
you to connect with other users of the project, learn about OTel, and provide

> feedback about your OTel usage:

Reviewer: Sounds a bit like summoning a deity to me. :)

How about "If you're looking for a vendor-agnostic space to..."

I don't think users are concerned with how this came to happen, but at the end you could link the working group as the source of this initiative, or for learning more.

Author: You're right, I felt it was a bit weird too but was going with what was already written... I rewrote it! Hopefully it reads better now, lmk what you think!

**Is this group only for OpenTelemetry end users?**

> No! Anyone is welcome to join and discuss their journey to observability. This

Reviewer: ```suggestion
Everybody is welcome to join and discuss their journey to observability. This
```
